composer.json

```diff
@@ -41,7 +18,7 @@
     "wpackagist-plugin/pantheon-advanced-page-cache": "^0.1.4",
     "wpackagist-plugin/wp-cfm": "^1.4.5",
     "wpackagist-theme/twentyseventeen": "^1.1",
-    "pantheon-systems/wordpress": "^4.8",
+    "pantheon-systems/wordpress-composer": "dev-master",
```

Comment:
We **should** be manually bumping up the version constraint on major version releases, since major version upgrades often break existing tests / clients. The composer.json should document which major version we tested with.

So, while I continue to disagree, if I have not convinced you by now I will concede the point to you -- but only because there will never be a commit made to `pantheon-systems/wordpress-composer` that is not tagged as a stable release, and no commit will be made there without some level of testing.